data_model.md

```diff
@@ -364,12 +377,12 @@ timestamp,value,quality_flag
       },
       “values”: [
         {
-          “timestamp”: “2018-11-22T12:01:48Z”,
+          “timestamp”: “2018-11-22T12:00:00Z”,
```

Comment:
should I replace all of the fancy quotes with plain quotes? while I don't really care for formatting, I'm worried that someone might copy/paste the template and end up eventually submitting fancy quotes that get rejected by the api.

Reply:
Sure. It definitely does get rejected w/ fancy quotes and not with regular